libr/core/cmd_search.c

```diff
@@ -2716,7 +2723,7 @@ static int cmd_search(void *data, const char *input) {
 		if (input[0] == 'j') {
 			json = true;
 		}
-	/* pass-thru */
+		// fallthrough
```

Comment:
The reason for that comment is to avoid an error in scan.coverity not sure if that change will be also accepted by them but we can try